tests/read-only-store.sh

```diff
@@ -0,0 +1,40 @@
+source common.sh
+
+enableFeatures "read-only-local-store"
+
```

Comment:
```suggestion

needLocalStore "cannot open store read-only when daemon has already opened it writeable"

```

According to @ericson2314 this should (hopefully) fix the test